docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -89,6 +89,21 @@ For more details on the Keycloak upgrade path, you can also read the [Bitnami Ke
 
 ## Version update instructions
 
+### v8.2.3
+
+:::caution Breaking Change
+
+Zeebe Gateway authentication is now enabled by default. To authenticate,
+
+1. [Create a client credential](/docs/guides/setup-client-connection-credentials.md)
+2. [Assign permissions to the application](/docs/self-managed/identity/user-guide/authorizations/managing-resource-authorizations.md)
+3. Then input the client credentials to your client. For example:
+
+- For connecting to the Desktop Modeler, look [here](/docs/components/modeler/desktop-modeler/connect-to-camunda-cloud.md).
```

Comment:
for better readability, it's [recommended](https://www.uow.edu.au/web-accessibility/web-accessibility-reference-site/what-you-can-do/links/) to use meaningful link text instead of "here".

Reply:
Definitely! Great feedback